entity-framework/core/managing-schemas/migrations/index.md

```diff
@@ -2,18 +2,35 @@
 title: Migrations - EF Core
 author: bricelam
 ms.author: bricelam
-ms.date: 10/30/2017
+ms.date: 10/05/2018
 uid: core/managing-schemas/migrations/index
 ---
 Migrations
 ==========
-Migrations provide a way to incrementally apply schema changes to the database to keep it in sync with your EF Core
-model while preserving existing data in the database.
 
-Creating the database
----------------------
-After you've [defined your initial model][1], it's time to create the database. To do this, add an initial migration.
-Install the [EF Core Tools][2] and run the appropriate command.
+A data model changes during development and gets out of sync with the database. You can drop the database and let EF create a new one that matches the model, but this procedure results in the loss of data. The EF Core Migrations feature preserves data by incrementally updating the database schema instead of creating a new database.
+
+Migrations includes command-line tools and APIs that help with the following tasks:
+
+* Create a migration. Generate code that can update the database to sync it with a set of model changes.
```

Comment:
Suggestion:

> * Create a migration. Generates code that represents a set of schema changes to be applied on the database.

Also, would we normally use ':' or emphasis for each of this, e.g.:

> **Create a migration:** Generates code that represents a set of schema changes to be applied on the database.

Just curious if there is a docs-specific practice I should be following elsewhere.

Reply:
I've seen it done various ways and haven't seen any particular way presented as normative.  In this case I agree it would look better with the first part emphasized in some way.  Since each bullet corresponds to an H2 by that title, I'll change them to internal links.